README.md

```diff
@@ -102,12 +102,21 @@ Such a build requires the presence of [CUnit](http://cunit.sourceforge.net/).  Y
 yourself, or you can choose to instead rely on the [Conan](https://conan.io) packaging system that
 the CI build infrastructure also uses.  In that case, install Conan and do:
 
-    $ conan install ..
+    $ conan install .. --build missing
+
+in the build directory prior to running cmake.
+
+The CUnit Conan package is hosted in the [Bintray repository](https://bintray.com/bincrafters/public-conan).
+In case this repository was not added to your Conan remotes list yet (and the above mentioned install
+command failed because it could not find the CUnit package), you can add the Bintray repository by:
+
+    $ conan remote add <REMOTE> https://api.bintray.com/conan/bincrafters/public-conan
```

Comment:
Perhaps better to just use `bincrafters` for a remote name? the `conanfile.txt` mentions that one explicitly.